cluster-scope/base/apiextensions.k8s.io/customresourcedefinitions/reports.api.curatoroperator.io/customresourcedefinition.yaml

```diff
@@ -0,0 +1,96 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.8.0
+  name: reportapis.curator.operatefirst.io
```

Comment:
The convention for naming of for CRDs is that the name of the custom resource definition should be the same as the name of the file.

Reply:
Updated for all the crds.